docs/config.md

```diff
@@ -62,15 +62,16 @@ for more details.
 
 ## Instrumentations
 
```

Comment:
```suggestion
## Instrumentations

All instrumentations are enabled by default  for all signal types (traces, metrics, and logs). 

You can disable all instrumentations for a specific signal type by setting the `OTEL_DOTNET_AUTO_{SIGNAL}_ENABLED_INSTRUMENTATIONS` environment variable to `false`.

For a more granular approach, you can disable specific instrumentations for a given signal type by setting the `OTEL_DOTNET_AUTO_{SIGNAL}_{0}_INSTRUMENTATION_DISABLED` environment variable to `false`, where `{SIGNAL}` is the type of signal, for example traces, and `{0}` is the case-sensitive name of the instrumentation.

```

Reply:
Applied in d4398104102607006798baa30f67b8d22d612dba